crates.json

```diff
@@ -2,199 +2,318 @@
   "boards": {
     "arduino_mkr1000": {
       "tier": 2,
-      "build": "cargo build --examples --all-features"
+      "build": "cargo build --examples --all-features",
+      "target": "thumbv6m-none-eabi"
```

Comment:
It's a bummer we need to specify the target here too since it is also in the cargo.toml of the BSP, right?

Reply:
I just put it there for convenience. It prevents us from installing both targets for a board that's only going to be using one or the other

Reply:
Is it possible to parse from the cargo.toml? Agreed it's good to only install the one that's needed. If it's difficult not a problem to keep things as is